.claude/commands/tests/test_pr_comment_formatter.py

```
@@ -0,0 +1,461 @@
+
+from pr_comment_formatter import (
+    CommentStatus,
+    CopilotComment,
+    PRCommentFormatter,
+    PRCommentResponse,
+    TaskItem,
+    UserComment,
+)
+
+# Import commentreply validate_comment_data - handle missing gracefully in logic
+from commentreply import validate_comment_data
```

Ruff (0.12.2)

Module level import not at top of file

(E402)